### PR TITLE
SAK-30007 Site tab name doesn't obey the user preferences to display …

### PR DESCRIPTION
**Pull request by juanjmerono**
…long or short description.

### CODE REVIEW

**Comment by juanjmerono** [comment]
I'm planning on merge this to fix this regression, ASAP, comments?

**Comment by ern** [comment]
Looks good

File: portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/SiteHandler.java [hunk 1]
```diff
@@ -850,6 +850,24 @@ public void includeTabs(PortalRenderContext rcontext, HttpServletRequest req,
...
+			{
+				Preferences prefs = PreferencesService.getPreferences(session.getUserId());
+				ResourceProperties props = prefs.getProperties("sakai:portal:sitenav");
+				try 
+				{
+					tabDisplayLabel = (int) props.getLongProperty("tab:label");
+				} 
+				catch (Exception any) 
+				{
+
```
Comment (ern) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/SiteHandler.java]: Lets not leave an empty catch statement, I would set it to some default here.

**Comment by juanjmerono** [comment]
Agree, I was just recovering the code removed previously, but sounds reasonable to add default value in catch block, fixed and ready to merge !!

**Pull request merged**